### DISCUSSION TITLE
Would be great if we Have "Access Informations" of a Field in GraphQL

### DISCUSSION THREAD

**Discussion by Temkit** [keystonejs/keystone#5329] [Feature requests]
Hi, it would be great if we can access the "access information" of a field,

so now we can do it for lists like this :

<img width="214" alt="Capture d’écran 2021-02-12 à 14 14 40" src="https://user-images.githubusercontent.com/13275723/107772593-b6f66b00-6d3c-11eb-88c1-fca5dc9570c6.png">

it would be great if we can have the same thing on Fields, since we can add access control on fields :

<img width="611" alt="Capture d’écran 2021-02-12 à 14 11 37" src="https://user-images.githubusercontent.com/13275723/107772364-73036600-6d3c-11eb-9681-5ea48e718c6e.png">

**Comment by bladey** [comment]
Thanks for the suggestion @Temkit!
